tests/integration/common/test_enricher.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from common.enricher import Enricher
+
+
+@pytest.fixture
+def enricher():
+    return Enricher()
+
+
+@pytest.mark.vcr
+def test_get_arxiv_categories_arxiv_id(enricher):
+    assert list(set(["hep-th", "hep-ph"])) == enricher._get_arxiv_categories(
+        arxiv_id="2112.01211"
+    )
+
+
+@pytest.mark.vcr
+def test_get_arxiv_categories_title(enricher):
+    assert ["hep-ex"] == enricher._get_arxiv_categories(
+        title="A strategy for a general search for new phenomena using data-derived signal regions and its application within the ATLAS experiment"
+    )
+
+
+def test_enricher(enricher: Enricher):
+    input_article = {"arxiv_eprints": [{"value": "2112.01211"}]}
+    assert {
+        "arxiv_eprints": [
+            {"value": "2112.01211", "categories": list(set(["hep-th", "hep-ph"]))}
```

Comment:
why do we need the set here?

Reply:
It's because of the issue with lists order assertion. For other tests I sorted the lists